security-claims/src/violation.rs

```diff
@@ -1,11 +1,35 @@
+use crate::{Claim, ClaimCipher, ClaimType};
```

Comment:
I'm not sure if its good that `violation.rs` is part of `security-claims`. The violation check is part of the fuzzer I think. The `security-claims` package only exists because of the header file and the custom `build.rs` script which I did not want to include in the main fuzzer.